distributed-databases/pinot/locals.tf

```diff
@@ -0,0 +1,15 @@
+#---------------------------------------------------------------
```

Comment:
remove this file and move this content to top of the `main.tf` file

Reply:
Done